modules/rtbhouseBidAdapter.js

```diff
@@ -96,30 +63,257 @@ export const spec = {
+ */
+function mapNativeAssets(slot) {
+  const params = slot.nativeParams || utils.deepAccess(slot, 'mediaTypes.native');
+  const assets = [];
+  if (params.title) {
+    assets.push({
+      id: OPENRTB.NATIVE.ASSET_ID.TITLE,
+      required: params.title.required ? 1 : 0,
+      title: {
+        len: params.title.len || 140
+      }
+    })
+  }
+  if (params.image) {
+    assets.push({
+      id: OPENRTB.NATIVE.ASSET_ID.IMAGE,
+      required: params.image.required ? 1 : 0,
+      img: mapNativeImage(params.image, OPENRTB.NATIVE.IMAGE_TYPE.MAIN)
+    })
+  }
+  if (params.icon) {
+    assets.push({
+      id: OPENRTB.NATIVE.ASSET_ID.ICON,
+      required: params.icon.required ? 1 : 0,
+      img: mapNativeImage(params.icon, OPENRTB.NATIVE.IMAGE_TYPE.ICON)
+    })
+  }
+  if (params.sponsoredBy) {
+    assets.push({
+      id: OPENRTB.NATIVE.ASSET_ID.SPONSORED,
+      required: params.sponsoredBy.required ? 1 : 0,
+      data: {
+        type: OPENRTB.NATIVE.DATA_ASSET_TYPE.SPONSORED
+      }
+    })
+  }
+  if (params.body) {
+    assets.push({
+      id: OPENRTB.NATIVE.ASSET_ID.BODY,
+      required: params.body.request ? 1 : 0,
+      data: {
+        type: OPENRTB.NATIVE.DATA_ASSET_TYPE.DESC,
+        len: params.body.len || undefined
+      }
+    })
+  }
+  if (params.clickUrl) {
+    assets.push({
+      id: OPENRTB.NATIVE.ASSET_ID.CLICK_URL,
+      required: params.clickUrl.required ? 1 : 0,
+      data: {
+        type: OPENRTB.NATIVE.DATA_ASSET_TYPE.CLICK_URL
+      }
+    })
+  }
+  if (params.cta) {
+    assets.push({
+      id: OPENRTB.NATIVE.ASSET_ID.CTA,
+      required: params.cta.required ? 1 : 0,
+      data: {
+        type: OPENRTB.NATIVE.DATA_ASSET_TYPE.CTA_TEXT,
+        len: params.cta.len || undefined
+      }
+    })
+  }
+  return assets;
+}
+
+/**
+ * Produces an OpenRTB Native Ads Image object
+ */
+function mapNativeImage(image, type) {
+  const img = {type: type};
+  if (image.aspect_ratios) {
+    const ratio = image.aspect_ratios[0];
+    const minWidth = ratio.min_width || 100;
+    img.wmin = minWidth;
+    img.hmin = (minWidth / ratio.ratio_width * ratio.ratio_height);
+  }
+  if (image.sizes) {
+    const size = Array.isArray(image.sizes[0]) ? image.sizes[0] : image.sizes;
+    img.w = size[0];
+    img.h = size[1];
+  }
+  return img
+}
+
+/**
+ * Produces Prebid bidObject from OpenRTB Bid object
+ */
+function interpretBannerBid(serverBid) {
+  return {
+    requestId: serverBid.impid,
+    mediaType: BANNER,
+    cpm: serverBid.price,
+    creativeId: serverBid.adid,
+    ad: serverBid.adm,
+    width: serverBid.w,
+    height: serverBid.h,
+    ttl: 55,
+    netRevenue: true,
+    currency: 'USD'
+  }
+}
+
+/**
+ * Produces Prebid bidObject from OpenRTB Bid object
+ */
+function interpretNativeBid(serverBid) {
+  return {
+    requestId: serverBid.impid,
+    mediaType: NATIVE,
+    cpm: serverBid.price,
+    creativeId: serverBid.adid,
+    width: 1,
+    height: 1,
+    ttl: 55,
```

Comment:
Maybe it's time to CONST the ttl for both? 
Probably we want the same time in both cases

Reply:
Fixed.